Added SeiManga extension

Teashrock opened the pull request:
Closes #1849. 

Checklist:

- [X] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [X] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [X] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio
- [X] Have removed `web_hi_res_512.png` when adding a new extension

Teashrock committed:
Added SeiManga extension

AwkwardPeak7 commented on src/ru/seimanga/build.gradle:
```diff
@@ -0,0 +1,10 @@
+    extClass = '.SeiManga'
+    themePkg = 'grouple'
+    baseUrl = 'https://seimanga.me'
+    overrideVersionCode = 46
```
```suggestion
    overrideVersionCode = 0
```

AwkwardPeak7 commented on src/ru/seimanga/src/eu/kanade/tachiyomi/extension/ru/seimanga/SeiManga.kt:
```diff
@@ -0,0 +1,199 @@
+
+class SeiManga : GroupLe("SeiManga", "https://seimanga.me", "ru") {
+
+    override val id: Long = 6
```
```suggestion
```
id should only be overriden if old one needs to be preserved. let it generate automatically

Teashrock committed:
Changed overrideVersionCode in build.gradle to 0

Co-authored-by: AwkwardPeak7 <48650614+AwkwardPeak7@users.noreply.github.com>

Teashrock committed:
Omitting id override from SeiManga.kt

Co-authored-by: AwkwardPeak7 <48650614+AwkwardPeak7@users.noreply.github.com>

Teashrock committed:
Removed a "needless blank line" because overpedantic Kotlin compiler said so

Seriously, what compiler in the world would even consider a blank line an error?

AwkwardPeak7 commented on src/ru/seimanga/src/eu/kanade/tachiyomi/extension/ru/seimanga/SeiManga.kt:
```diff
@@ -0,0 +1,197 @@
+                } catch (e: Exception) {
+                    e.printStackTrace()
+                    false
+                }
```
```suggestion
                Toast.makeText(screen.context, "Для смены домена необходимо перезапустить приложение с полной остановкой.", Toast.LENGTH_LONG).show()
                true
```
build failed here btw, also you don't need to manually set the preference value, app will take care of it.

Teashrock replied:
What's going on? The reason of the error is simple: it changed "commit" function name to "memit" for some reason. The latter function doesn't exist. What's even stranger is that everything's fine in my edited code, but not in the commit.

AwkwardPeak7 commented on src/ru/seimanga/src/eu/kanade/tachiyomi/extension/ru/seimanga/SeiManga.kt:
```diff
@@ -0,0 +1,197 @@
+    override fun setupPreferenceScreen(screen: androidx.preference.PreferenceScreen) {
+        super.setupPreferenceScreen(screen)
+        EditTextPreference(screen.context).apply {
+            key = DOMAIN_TITLE
```
while you are at it, change the domain preference implementation to [something like this](https://github.com/keiyoushi/extensions-source/blob/fea4ac2a754fc1d03b43eaaacb544854ba2a4865/src/en/hiperdex/src/eu/kanade/tachiyomi/extension/en/hiperdex/Hiperdex.kt#L40-L68) so domain update in extension code change the pref for users too

AwkwardPeak7 reviewed (changes requested):
> Seriously, what compiler in the world would even consider a blank line an error?

Thats the linter for you 😂

AwkwardPeak7 committed:
Update SeiManga.kt

AwkwardPeak7 committed:
Update GroupLe.kt

AwkwardPeak7 committed:
Update build.gradle.kts

AwkwardPeak7 committed:
Update AllHentai.kt

AwkwardPeak7 committed:
Update MintManga.kt

AwkwardPeak7 committed:
Update AllHentai.kt

AwkwardPeak7 committed:
Update SeiManga.kt

AwkwardPeak7 committed:
Update build.gradle

AwkwardPeak7 committed:
Update build.gradle
